rtm/plotting.py

```diff
@@ -223,6 +232,10 @@ def plot_time_slice(S, processed_st, time_slice=None, label_stations=True,
     if dem is not None:
         ax.set_aspect('equal')
 
+    # crop plot to show just the slice area
```

Comment:
Nothing funky with regional example for this? Would be good to check

Reply:
> I think this idea makes sense. Along with what you already said, it's nice to remove the scalebar. However, I think it'd be nicer to have the origin be the grid center, i.e. the grid center is at (0, 0). Then it's easy to see how far from the grid center things are. I also left a few more comments on the code itself.

At first I wasn't sure, but I think I like it. Easy change.

Reply:
Regional example looks funky for me with hires=True. Looks fine w/ hires=False. Can you check yours? This is irrespective of this branch.

Reply:
Oh, I see now. Only used if `xy_plot` is not `None`.

Reply:
Yuck. Looks bad for me too. Will investigate. Resolving this though.

Reply:
Fixed with 769d3ca on this PR